src/dotnet/APIView/APIViewWeb/Repositories/CosmosPullRequestsRepository.cs

```diff
@@ -52,7 +54,22 @@ private async Task<List<PullRequestModel>> GetPullRequestFromQueryAsync(string q
                 var result = await itemQueryIterator.ReadNextAsync();
                 allRequests.AddRange(result.Resource);
             }
-            return allRequests;
+
+            // Cosmos doesn't allow cross join of two containers so we need to filter closed API reviews

```

Comment:
Are we worried about this becoming slow over time? If so perhaps we could create a view or projection of some sort on the db side.

Reply:
Long term solution is to add a status field in pull request model entry as well or a trigger in the cosmos side.